### PR TITLE
[merge jam] port #49949

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Fix issues with startup in Windows

### What issues does this PR fix or reference?
Merges #49901 into master branch

### CODE REVIEW

**Review by DmitryKuzmenko — DISMISSED** [review]

File: salt/grains/core.py [hunk 1]
```diff
@@ -2697,13 +2694,13 @@ def _hw_data(osdata):
         # On SmartOS (possibly SunOS also) smbios only works in the global zone
         # smbios is also not compatible with linux's smbios (smbios -s = print summarized)
         grains = {
-            "biosversion": __salt__["smbios.get"]("bios-version"),
+            "biosversion": __salt_gg_["smbios.get"]("bios-version"),
```
Comment (DmitryKuzmenko) [inline salt/grains/core.py:2697]: `__salt_gg_` is a mistype?

**Review by DmitryKuzmenko — CHANGES_REQUESTED** [review]
Found just one mistype. Other things are good.

**Review by DmitryKuzmenko — APPROVED** [review]

File: salt/grains/core.py (hunk 1 above)
Comment (DmitryKuzmenko) [reply salt/grains/core.py]: I've updated it.

**Comment by twangboy** [comment]
@DmitryKuzmenko Thanks for catching that!

**Pull request merged**